daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/transaction/CheckAuthorization.scala

```diff
@@ -86,20 +86,10 @@ private[lf] object CheckAuthorization {
   private[lf] def authorizeExercise(ex: ExercisesContext)(
       auth: Authorize,
   ): List[FailedAuthorization] = {
-    val controllersDifferFromActors = ex.controllers != ex.actingParties
     authorize(
       passIf = ex.actingParties.nonEmpty,
       failWith = FailedAuthorization.NoControllers(ex.templateId, ex.choiceId, ex.optLocation),
     ) ++
-      authorize(
-        passIf = !controllersDifferFromActors,
-        failWith = FailedAuthorization.ActorMismatch(
```

Comment:
We can drop the `ActorMismatch` class now as well, can't we?

Reply:
good point.